### ISSUE TITLE
Icon request: icon-tabs. 

### ISSUE THREAD

**Issue by diosney** [FortAwesome/Font-Awesome#2419]
An icons that resembles a tab component.
![icon-tabs](https://f.cloud.github.com/assets/1340074/1551154/93a6e782-4e0f-11e3-8b2b-d3b8cb1ff015.png)

**Comment by mikethedj4** [comment]
+1 

![tabs-icon](https://cloud.githubusercontent.com/assets/2473707/5607248/da9b32c4-9418-11e4-9b0c-4b9b1d0e9e21.png)

**Comment by rex** [comment]
Major +1 :arrow_up: for this as well! Seems a very simple icon in its most basic form and would be helpful for many interface building tools!

**Comment by benediktfrings** [comment]
+1

**Comment by remypar5** [comment]
+1

**Comment by tituswoo** [comment]
+1

**Comment by eric802222** [comment]
+1

**Comment by feitzi** [comment]
+1

**Comment by mustardBees** [comment]
+1

**Comment by joseph-sm** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Hey there!

We've made the decision to move our icon requests flow to **GitHub Discussions**, as opposed to issue tracking. No more +1s or thumbs-ups needed. There's an upvote system built in.

As part of this transition, we are closing existing icon requests with lower comment or reaction counts, but **we are not saying no** to your request. (We know some of these have been open since before I had gray hair. And I've had gray hair for a LONG time.)

**Could you do us a favor? If this icon request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
→ https://github.com/FortAwesome/Font-Awesome/discussions/new?category=icon-requests

And listen. We know this is annoying. We would batch convert them to discussions if we could, but Github's removed that ability earlier this year. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worries at all. I love doing things twice.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. This request already HAS an auto-response saying that you're refining the workflow. What do you mean you're refining your refinement? That's a bunch of bull-phooey.
* No biggie. Thanks for letting us know what's up.

**Comment by t3rminus** [comment]
> If an issue closes after seven years and nobody responds, did it ever really exist?

Seven years is optimistic. This issue is over 12 years old.

Project I needed this icon for was made without it. The project lasted many years, and was then shutdown, and has been shutdown for many more years.

If this icon wasn’t made/added in 12 years, moving it to a GitHub discussion is not going to change that.

It’s too little, too late for an icon pack that killed its community goodwill, became technically obsolete, became visually stale, and lost any chances of me deciding to use it for future projects. I guess the only good thing to come out of this is I’ll no longer get pings from more people +1-ing this issue in my inbox.

In summary, no biggie. Thanks for letting us know what's up.

**Comment by sensibleworld** [comment]
@t3rminus Thanks for sharing your perspective, and sorry we didn’t get to this in a timeframe that would have helped your earlier project. Our icon design team is quite small, and we’re working on cleaning up and improving our workflow so newer requests don’t fall through the cracks the same way.

Wishing you well on whatever you’re building next.
